vnext/Microsoft.ReactNative/Views/SwitchViewManager.cpp

```diff
@@ -42,12 +42,58 @@ class SwitchShadowNode : public ShadowNodeBase {
   winrt::Microsoft::ReactNative::JSValue m_onTrackColor;
 };
 
+// sets the resources to SolidColorBrushes if the user sets the color before runtime, this lets the user change the
+// color of the SolidColorBrush at runtime and keeps the toggleswitches who did not change colors on the default
+// ThemeResource
+void setBrushes(
+    const winrt::Microsoft::ReactNative::JSValueObject &props,
+    const xaml::Controls::ToggleSwitch toggleSwitch) {
+  for (const auto &pair : props) {
+    const std::string &propertyName = pair.first;
+    const auto &propertyValue = pair.second;
+    if (propertyName == "thumbTintColor") {
+      toggleSwitch.Resources().Insert(winrt::box_value(L"ToggleSwitchKnobFillOff"), xaml::Media::SolidColorBrush());
+      toggleSwitch.Resources().Insert(
+          winrt::box_value(L"ToggleSwitchKnobFillOffPointerOver"), xaml::Media::SolidColorBrush());
+      toggleSwitch.Resources().Insert(
+          winrt::box_value(L"ToggleSwitchKnobFillOffPressed"), xaml::Media::SolidColorBrush());
+      toggleSwitch.Resources().Insert(
+          winrt::box_value(L"ToggleSwitchKnobFillOffDisabled"), xaml::Media::SolidColorBrush());
+      toggleSwitch.Resources().Insert(
+          winrt::box_value(L"ToggleSwitchKnobFillOnPointerOver"), xaml::Media::SolidColorBrush());
+      toggleSwitch.Resources().Insert(
+          winrt::box_value(L"ToggleSwitchKnobFillOnDisabled"), xaml::Media::SolidColorBrush());
+      toggleSwitch.Resources().Insert(winrt::box_value(L"ToggleSwitchKnobFillOn"), xaml::Media::SolidColorBrush());
+      toggleSwitch.Resources().Insert(
+          winrt::box_value(L"ToggleSwitchKnobFillOnPressed"), xaml::Media::SolidColorBrush());
```

Comment:
I'm not following what this is doing, this is setting an empty brush on these resource keys, and isn't using propertyValue at all

Reply:
Yeah! This was my solution for overwriting the SystemThemeResourceBrush (which I think it the right term?) that the ToggleSwitch's Resources gets mapped to by default. After these get updated, the function UpdateToggleResourceBrush is able to fill the color property of SolidColorBrush. 

If I don't set these, the function UpdateToggleResourceBrush updates SystemThemeResourceBrush which changes the color of **ALL** ToggleSwitches instead of just one. I couldn't find a function that would check if a SolidColorBrush was a SystemThemeResourceBrush but if you know one, I can add that in!

Reply:
I _suspect_ that this change will break theme changing - i.e. run the app, show toggleswitches, then change the system/app theme (dark/light) - the toggleswitches should render correctly (i.e. they should change color)

Reply:
This may be ok as long as controls that don't customize these properties still respond to theme changes. It looks like that's the case but please verify :)

Reply:
yeah, I actually did have the problem in development but adding the if statements made sure that we only overwrite the properties that change colors! Just double checked against main and it has the same output when switching themes (you can also see this at the very end of _after changes_ video).